_includes/android/files.md

```diff
@@ -13,8 +13,8 @@ ParseFile file = new ParseFile("resume.txt", data);
 
 Notice in this example that we give the file a name of `resume.txt`. There's two things to note here:
 
-*   You don't need to worry about filename collisions. Each upload gets a unique identifier so there's no problem with uploading multiple files named `resume.txt`.
-*   It's important that you give a name to the file that has a file extension. This lets Parse figure out the file type and handle it accordingly. So, if you're storing PNG images, make sure your filename ends with `.png`.
+*   You don't need to worry about file name collisions. Each upload gets a unique identifier so there's no problem with uploading multiple files named `resume.txt`.
```

Comment:
Is `file name` an American English thing? - my Oxford (British) English dictionary only recognises `filename`, although amusingly autocorrect really wants me to write `file name` 😅

Reply:
As far as I can tell, I use British english too so not sure. I ran the MD files through a [spell check](https://github.com/lukeapage/node-markdown-spellcheck) using en-US and these are effectively the results. Might need to get clarification.

Reply:
I checked the Oxford American dictionary and it does recognise 'filename' so I'm inclined to use that.